### ISSUE TITLE
Clojure deref (@) is flagged as "err"

### ISSUE THREAD

**Issue by jszakmeister** [jneen/rouge#583]
I've been trying to convert my website to Jekyll, and a snippet of code I have is rendering with an error because of the deref (@) in the code.  Here's the snippet of code:

```clojure
(defn d3-gauge [args ...]
  (let [dom-node (r/atom nil)]
    (r/create-class
      {:component-did-update
       (fn [this old-argv]
         (let [[_ args ...] (r/argv this)]
           ;; This is where we get to actually draw the D3 gauge.
           (draw-d3-gauge @dom-node args ...)))

       :component-did-mount
       (fn [this]
         (let [node (r/dom-node this)]
           ;; This will trigger a re-render of the component.
           (reset! dom-node node)))

       :reagent-render
       (fn [args ...]
         ;; Necessary for Reagent to see that we depend on the dom-node r/atom.
         ;; Note: we don't actually use any of the args here.  This is because
         ;; we cannot render D3 at this point.  We have to wait for the update.
         @dom-node
         [:div.gauge [:svg]])})))
```

Which contains the following in the final render:

```html
<span class="err">@</span><span class="n">dom-node</span>
```

The "err" is the part that is important here.  The highlighter is incorrect--it's valid Clojure code.

**Comment by jneen** [comment]
Hi! I think I've fixed this in 14b4b1e - could you confirm?

**Comment by jszakmeister** [comment]
Thank you for fixing it.  I'm unsure how to test this in my Jekyll environment, but going through rougify it does appear that the problem is fixed.  Thank you!
